### ISSUE TITLE
Sandbox is verbose logging even when runtime logging is disabled

### ISSUE THREAD

**Issue by twschiller** [pixiebrix/pixiebrix-extension#7159]
![image](https://github.com/pixiebrix/pixiebrix-extension/assets/1879821/0f712c48-d77b-45d8-97ad-83d4f1cd5822)

**Comment by fregante** [comment]
Yes because the sandbox doesn't have access to chrome.storage. Given it's a sandbox I'm not sure it has access to the same localStorage either. 

If it isn't already, we can only toggle it via ENV

**Comment by fregante** [comment]
https://github.com/pixiebrix/pixiebrix-extension/blob/c9d5f99cc6ef7e331c87f8699e32a703aab869e7/src/utils/postMessage.ts#L43

I'll restore the ENV check
